scripts/jenkins/ardana/gerrit/gerrit_handle_event.py

```diff
@@ -0,0 +1,166 @@
+#!/usr/bin/env python
+
+import argparse
+import os
+import sys
+
+sys.path.append(os.path.dirname(__file__))
+from gerrit import GerritChange, GerritChangeSet  # noqa: E402
+from gerrit_merge import gerrit_merge  # noqa: E402
+from gerrit_review import gerrit_review  # noqa: E402
+
+
+def get_submittable_references(merged_change):
+    """
+    Get all submittable changes that have an explicit Depends-On
+    referencing the indicated merged Gerrit change and targeting the
+    same branch.
+
+    :param merged_change: GerritChange object representing a merged change
+    :return: list of GerritChange object representing submittable changes,
+    referencing the supplied merged change as a dependency, and which could
+    be merged themselves
+    """
+
+    changeset = GerritChangeSet(
+        'is:open',
+        'is:mergeable',
+        'branch:{}'.format(merged_change.branch),
+        'message:"Depends-On:"',
+        'label:Code-Review=2',
+        'label:Verified=2',
+        'label:Workflow=1',
+        '-label:Verified=-1',
+        '-label:Code-Review=-2'
+    )
+
+    references = []
+    for change in changeset.changes():
+        if change.submittable and change.depends_on(merged_change):
+            references.append(change)
+
+    return references
+
+
+def get_stale_references(updated_change):
+    """
+    Get all changes that have an explicit Depends-On directly or indirectly
```

Comment:
(nit) it's not really explicit when it's indirectly ;-)

Reply:
right, I updated the description